definitions/json/AuthorisationCaseField.json

```diff
@@ -2459,6 +2459,54 @@
     "UserRole": "et-acas-api",
     "CRUD": "R"
   },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "claimantDocumentCollection",
+    "UserRole": "caseworker-employment-legalrep-solicitor",
+    "CRUD": "CRU"
```

Comment:
Legal reps won't have access to edit this anyway but this might be better off as R only.

Reply:
done